browser/BUILD.gn

```diff
@@ -190,8 +196,28 @@ source_set("browser_process") {
       "android/preferences/brave_pref_service_bridge.cc",
       "android/preferences/website/desktop_mode_tab_helper.cc",
       "android/preferences/website/desktop_mode_tab_helper.h",
+      "android/rewards/safetynet_check.cc",
+      "android/rewards/safetynet_check.h",
     ]
 
+    if (brave_rewards_enabled) {
+      sources += [
+        "android/rewards/brave_rewards_native_worker.cc",
+        "android/rewards/brave_rewards_native_worker.h",
+      ]
+
+      deps += [
+        "//brave/vendor/bat-native-ledger",
+      ]
+
+      configs += [ ":safetynet_config" ]
+    }
+    if (brave_ads_enabled) {
+      sources += [
+        "android/rewards/brave_ads_native_helper.cc",
```

Comment:
same as above for path, we should have naming and path consistency

Reply:
There is no such file right now, it will be added when ads flag is enabled. I commented that part of code for now